src/engine/bank01.asm

```diff
@@ -2791,17 +2791,21 @@ CardListItemSelectionMenu:
 	ld a, [wCardListItemSelectionMenuType]
 	or a
 	ret z
-	ldtx hl, Text008b ; SelectCheckText
+	ldtx hl, SelectCheckText
 	ld a, [wCardListItemSelectionMenuType]
 	cp PLAY_CHECK
 	jr nz, .got_text
 	ldh a, [hTempCardIndex_ff98]
 	call LoadCardDataToBuffer1_FromDeckIndex
-	ldtx hl, Text008a ; PlayCheck2Text
+; handle verbs
+; redundant in English, where both are just "PLAY"
+; originally "だす" (to put out)
+	ldtx hl, PlayCheck2Text
 	ld a, [wLoadedCard1Type]
 	cp TYPE_TRAINER
 	jr nz, .got_text
-	ldtx hl, Text0089 ; PlayCheck1Text
+; originally "つかう" (to use)
+	ldtx hl, PlayCheck1Text
```

Comment:
Is it worth communicating this through the labels?

ie:
`PlayCheck1Text` -> `UseCheckText`
`PlayCheck2Text` -> `PutOutCheckText`

or whatever is most appropriate.